frontend/src/bundles/landing/pages/landing.tsx

```diff
@@ -51,8 +51,8 @@ const Landing = (): JSX.Element => {
         }
     }, [dispatch, user]);
 
-    const signUpHandler = useCallback((): void => {
-        dispatch(appActions.navigate(AppRoute.SIGN_UP));
+    const signInHandler = useCallback((): void => {
```

Comment:
naming should start with handle by quality criteria

Reply:
Thanks, fixed it